jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt

```diff
@@ -108,3 +115,21 @@ class ContainerLogsAction(
         return true
     }
 }
+
+class ExecuteCommandAction(
+    private val project: Project,
+    private val container: ContainerDetails
+) : AnAction(message("ecs.execute_command_run"), null, null) {
+    override fun actionPerformed(e: AnActionEvent) {
+        var sessionManagerInstalled: Int = 1
+        runBlocking(Dispatchers.IO) {
+            val process = CapturingProcessHandler(GeneralCommandLine("session-manager-plugin")).runProcess()
```

Comment:
can/should we cache this check somehow?

Reply:
Trying to see how we could implement this, seems like a good idea